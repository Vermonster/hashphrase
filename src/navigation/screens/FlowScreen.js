import React from 'react';
import { withNamespaces } from 'react-i18next';
import { View } from 'react-native';
import { Card, Button } from 'react-native-paper';

const FlowScreen = (props) => {
  const { t, navigation: { navigate } } = props;

  return (
    <View>
      <Card>
        <Button
          onPress={() => {
            navigate('PasswordScreen', {
              isNewPassword: false,
            });
          }}
          mode="contained"
        >
          {t('oldPasswordFlow')}
        </Button>
        <Button
          onPress={() => {
            navigate('PasswordScreen', {
              isNewPassword: true,
            });
          }}
          mode="contained"
        >
          {t('newPasswordFlow')}
        </Button>
      </Card>
    </View>
  );
};

export { FlowScreen as TestFlowScreen };
export default withNamespaces()(FlowScreen);
