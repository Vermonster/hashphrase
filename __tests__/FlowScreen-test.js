import React from 'react';
import { shallow } from 'enzyme';
import FlowScreen from '../src/navigation/screens/FlowScreen';

describe('<FlowScreen />', () => {
  it('should render the same snapshot', () => {
    const navigation = { navigate: jest.fn() };
    const wrapper = shallow(<FlowScreen navigation={navigation} />);
    expect(wrapper).toMatchSnapshot();
  });
});
