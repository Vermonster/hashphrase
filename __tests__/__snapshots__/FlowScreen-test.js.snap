// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FlowScreen /> should render the same snapshot 1`] = `
<ForwardRef(View)>
  <withTheme(Card)>
    <withTheme(Button)
      mode="contained"
      onPress={[Function]}
    >
<<<<<<< HEAD
      Old password flow
=======
      oldPasswordFlow
>>>>>>> update snaps
    </withTheme(Button)>
    <withTheme(Button)
      mode="contained"
      onPress={[Function]}
    >
<<<<<<< HEAD
      New password flow
=======
      newPasswordFlow
>>>>>>> update snaps
    </withTheme(Button)>
  </withTheme(Card)>
</ForwardRef(View)>
`;
